Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -153,15 +159,28 @@ public class RocksDBStore : BaseStore
         /// <inheritdoc/>
         public override IEnumerable<Guid> ListChainIds()
         {
-            return _liteDb.GetCollectionNames()
-                .Where(name => name.StartsWith(IndexColPrefix))
-                .Select(name => ParseChainId(name.Substring(IndexColPrefix.Length)));
+            string path = Path.Combine(_path, ChainDbName);
+
+            foreach (string name in RocksDb.ListColumnFamilies(_options, path))
+            {
+                Guid guid;
+
+                try
+                {
+                    guid = new Guid(name);
```

Comment:
According to [a micro benchmark conducted by someone at Stack Overflow][1], `Guid.Parse(string)` is slighly faster than `new Guid(string)`.

```suggestion
                    guid = Guid.Parse(name);
```

[1]: https://stackoverflow.com/a/17570268/383405